### PR TITLE
Update Uber

### PR DESCRIPTION
**Pull request by leggomyinfo**
I was using TOTP for Uber. I disabled it and re-enabled it in order to set up TOTP on a different authenticator app. Once re-enabled, SMS was the only option.

### CODE REVIEW

File: entries/u/uber.com.json [hunk 1]
```diff
@@ -3,13 +3,12 @@
     "domain": "uber.com",
     "url": "https://www.uber.com/",
```
Comment (hkamran80) [inline entries/u/uber.com.json]:
```suggestion
```

The `url` key is only needed if going to the domain (e.g. `https://uber.com`) fails

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by hkamran80** [comment]
It's interesting that they haven't updated their docs yet

**Comment by Carlgo11** [comment]
@leggomyinfo Have you reached out to Uber Support?
Their [help article](https://help.uber.com/riders/article/turn-on-2-step-verification?nodeId=b8bb9152-8c91-4f49-83c4-35cf2e1dcf72) on the matter still states TOTP is supported.

**Comment by leggomyinfo** [comment]
I just contacted support. I’ll let you know their reply.

**Comment by kmpoppe** [comment]
Pre-Pandemic I wasn't even able to log into my Uber account that I created for a tour in the UK, which nowadays seems to work.
I had set up TOTP in the account in 2019 and it is still working so I wouldn't remove TOTP as of now.

**Comment by leggomyinfo** [comment]
After a lot of back and forth, Uber support was totally unhelpful. They agreed to escalate the ticket a few days ago — still waiting on a reply.

The primary audience for 2fa.directory is people looking up a site/service to see if a 2FA method is available, not people who already have that 2FA method enabled for the site/service. If you accept that premise, and if TOTP is in fact only available to Uber users who already have it enabled, it makes more sense to remove it and explain the weird situation with the tooltip note. At a minimum, if TOTP remains checked off for Uber, there should be a tooltip note – I'm guessing everyone can agree on that.

If there is someone who is willing to risk losing TOTP for Uber or doesn't already have it, please test it by disabling/re-enabling or trying to enable TOTP for the first time.

**Comment by kmpoppe** [comment]
> After a lot of back and forth, Uber support was totally unhelpful.

Which was exactly my experience with them after debating why I wasn't able to access my account from Germany when I created it in the UK...

> At a minimum, if TOTP remains checked off for Uber, there should be a tooltip note – I'm guessing everyone can agree on that.

Indeed. With `notes` in the file, an exclamation mark is shown with could be used for warning users that TOTP seems to be unavailable for new setup as of a right now.

**Comment by Carlgo11** [comment]
I took it upon myself to verify and yes, it is indeed not an option after resetting it.
I think the note is redundant (and misspelled?) but support removing TOTP from the list.

File: entries/u/uber.com.json [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
     "tfa": [
-      "sms",
-      "totp"
+      "sms"
     ],
     "documentation": "https://help.uber.com/h/b8bb9152-8c91-4f49-83c4-35cf2e1dcf72",
-    "notes": "2FA can only be enabled via the app",
+    "notes": "2FA can only be enabled via the app. TOTP is no longer an option but grandathered in if previously set up.",
```
Comment (Carlgo11) [inline entries/u/uber.com.json]:
```suggestion
    "notes": "2FA can only be enabled via the app",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**